ssg/build_cpe.py

```diff
@@ -43,11 +43,13 @@ def load_product_cpes(self, env_yaml):
             product_cpes_list = env_yaml["cpes"]
             for cpe in product_cpes_list:
                 for cpe_id in cpe.keys():
+                    # these product CPEs defined in product.yml are defined
```

Comment:
Is this difference temporal, or will it be like that for a long time? I don't find the assignment of ID as something strange, so if there is a chance that it won't be needed in the future, I would suggest removing it.

Reply:
This should be only temporary. In the end, I would like to get rid of these CPE definitions in product.yaml entirely and replace it just with
```
platforms:
```
key like in rule.yml. This should be one of benefits of introducing version specification through CPE AL - getting rid of these definitions.